dist/pinny.js

```diff
@@ -35,7 +38,8 @@
     /* jshint ignore:end */
 
     var $window = $(window);
-    var iOS7orBelow = $.os.ios && $.os.major <= 7;
+    var needsSpacer = ($.os.ios && $.os.major <= 7) ||
+        ($.os.android && $.os.major <= 4);
```

Comment:
Something like `$.os.android && $.os.aosp && $.os.major <= 4` makes sense. But technically Chrome will never hit this condition because its major hasn't been single digit in a long long time.